sdk/docs/pages/typescript/sui-client.mdx

```diff
@@ -130,3 +130,31 @@ const client = new SuiClient({
 	}),
 });
 ```
+
+## Pagination
+
+`SuiClient` exposes a number of RPC methods that return paginated results. These methods return a
+result object with 3 fields:
+
+- `data`: The list of results for the current page
+- `nextCursor`: a cursor pointing to the next page of results
+- `hasNextPage`: a boolean indicating whether there are more pages of results
+
+Some APIs also accept an `order` option that be set to either `ascending` or `descending` to change
+the order in which the results are returned.
+
+The `nextCursor` can be passed to the `cursor` option of the RPC method to retrieve the next page,
```

Comment:
you can pass the `nextCursor` to the `cursor`
not code format on line 140 though